spec/providers/coreIntegration/tcpsocket.integration.src.js

```diff
@@ -99,5 +99,47 @@ module.exports = function (provider, setup) {
       });
   });
 
+  it("Pause and Resume work", function (done) {
+    socket.connect('www.google.com', 80, function () {
+      var paused = false;
+      var messageCount = 0;
+      dispatch.on('onMessage', function(msg) {
+        if (!('data' in msg)) {
+          // Not an 'onData' message.
+          return;
+        }
+
+        // One onData is allowed during pause due to https://crbug.com/403076.
+        ++messageCount;
+        if (messageCount === 1) {
```

Comment:
Not confident, no.  I've only tested with Chrome.  I've changed the logic here so that the test should pass if it receives only one message.